### PR TITLE
Go live w/ Reserve site re-design 

### PR DESCRIPTION
**Pull request by denviljclarke**
### Description

This PR merges dev into the main branch, adding all the re-design changes to the reserve site

### CODE REVIEW

**Review by olenovyk — APPROVED** [review]
LGTM!

**Review by chapati23 — CHANGES_REQUESTED** [review]
a few things that look off compared to design, screenshots posted in the previous PR that got merged while I was just posting them: https://github.com/mento-protocol/reserve-site/pull/99

File: src/components/ReserveComposition.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                    key={item.token}
+                    className="flex flex-row items-center justify-start"
+                  >
+                    <div
+                      className="mr-[10px] h-[18px] w-[18px] rounded"
+                      style={{ backgroundColor: TokenColor[item.token] }}
+                    />
+                    <div>
+                      <span className="font-fg text-[22px] font-medium">{`${item.percent.toFixed(2)}%`}</span>{" "}
+                      {item.token}
```
Comment (chapati23) [inline src/components/ReserveComposition.tsx:88]: token should also be 22px 

image

**Review by chapati23 — CHANGES_REQUESTED** [review]
last one i swear

**Review by RyRy79261 — APPROVED** [review]
LGTM based off @chapati23 's comments

**Review by chapati23 — APPROVED** [review]
let's go live!

**Pull request merged**